tasmota/xsns_02_analog.ino

```diff
@@ -36,6 +37,8 @@
 #define ANALOG_RANGE                  4095             // 4095 = 12, 2047 = 11, 1023 = 10
 #undef ANALOG_PERCENT
 #define ANALOG_PERCENT                ((ANALOG_RANGE + 50) / 100)  // approximation to 1% ADC range
+#undef ANALOG_BOARD
+#define ANALOG_BOARD                  "ESP-32"
```

Comment:
What is this #define ANALOG_BOARD used for ?

Reply:
I'm used library MQUnifiedsensor-2.0.1 but there it isn't used... I'm solving